websites.yml

```diff
@@ -89,8 +89,8 @@ avatar:
   github: opengraph
   gitlab: opengraph
   instagram: opengraph
-  openhub: false
-  opensuse: false
+  openhub: openhub
```

Comment:
It would be better if these were a generic solution, as there will be other websites where we need to same approach

```
  openhub:
    html_selector: '#account_icon > img'
  opensuse:
    html_selector: '#profile_icon_wrapper > img'
```